refinery/data_set_manager/utils.py

```diff
@@ -615,12 +613,8 @@ def _generate_solr_params(params, assay_uuids=[]):
                   'facet.limit=-1'
                   ])
 
-    solr_params = 'fq=({})'.format(
-        ' OR '.join(map(
-            lambda id: 'assay_uuid:{}'.format(id),
-            assay_uuids
-        ))
-    )
+    assert len(assay_uuids) > 0, 'At least one assay must be accessible'
```

Comment:
Nothing will go wrong at the next line if `assya_uuids == []`, except that the solr query constructed is invalid: so it seems best to catch it here. That said, perhaps what we really want is a custom error, that can be caught a couple layers up.